vendor/engines/secure_rooms/app/controllers/secure_rooms/secure_room_occupancies_controller.rb

```diff
@@ -0,0 +1,13 @@
+module SecureRooms
+
+  class SecureRoomOccupanciesController < ApplicationController
```

Comment:
It's not a big deal, but `SecureRooms::SecureRoomOccupanciesController` feels a little redundant. What was the main reason for naming it this?